### PR TITLE
HPC Module Maintenance

### PR DESCRIPTION
**Pull request by thecyclingcyclopse**
HPC Module Maintenance

index.md:
Capitalization of title
Fixed link to terminal module

### CODE REVIEW

File: content/en/modules/HPC/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The estimated time to complete this training module is 4h.
 
 The prerequisites to take this module are:
  * the [installation](/modules/installation) module.
- * the [introduction to the terminal](/modules/terminal) module.
+ * the [introduction to the terminal](/modules/introduction_to_terminal) module.
```
Comment (htwangtw) [inline content/en/modules/HPC/index.md:33]: if possible, can you build the website with the [instruction](https://github.com/school-brainhack/school-brainhack.github.io#run-locally) to verify the URL change is working?

**Review by htwangtw — COMMENTED** [review]
Thank you! Can you test the change to the link? If not let us know.

File: content/en/modules/HPC/index.md (hunk 1 above)
Comment (thecyclingcyclopse) [reply content/en/modules/HPC/index.md:33]: Yes, I will! Something did not work out as intended because the link still appears to be broken.

Comment (htwangtw) [reply content/en/modules/HPC/index.md:33]: I will need to build your changes on my side to check this so I will leave it until later....

Comment (me-pic) [reply content/en/modules/HPC/index.md:33]: I tested all the links, and it's working on my side !

**Pull request merged**